src/pages/guides/gateway/create-tenant.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Create a Tenant
+description: Create a configuration file for your tenants, access the gateway, and update your configuration file.
+---
+
+# Create a tenant
+
+1. Create and save a JSON configuration file that defines the properties of your mesh. Your mesh is defined by a combination of [handlers] and [transforms]. In this example, the file name is `mesh.json`. The `tenantId` is the case-sensitive name you want to assign to your mesh configuration.
+
+<InlineAlert variant="info" slots="text"/>
+
+The following example adds both an Adobe Commerce instance (with Live Search enabled) and an Adobe Experience Manager instance to the mesh. The GraphQL endpoints for Commerce and Live Search are different, therefore you must configure them separately.
+
+    ```json
```

Comment:
The code sample needs to be left justified. Also, the code gates need to be closed after line 61.

Reply:
it looks like this one needs to stay indented or it break numbering.